src/app/test/test.component.ts

```diff
@@ -287,12 +288,12 @@ export class TestComponent implements OnInit, AfterViewInit {
     this.reports.forEach((report) => (report.checked = false));
   }
 
```

Comment:
```suggestion
setUpdatePathAction(value: UpdatePathAction) {
  this.updatePathAction = value;
}
```

Reply:
I added the change, but is there a reason for using a separate function to do this instead of assigning it directly in the html? Or is it just for typesafety since we do a check here for UpdatePathAction type?

Reply:
I think its better to have the html just call functions